content/applications/finance/accounting/payables/pay/sepa.rst

```diff
@@ -2,135 +2,96 @@
-bank account and select the "Settings" option.
+Non-EEA territories:
 
-To activate SEPA, click the **Advanced Settings** tab and, in the **Payment
-Methods** part of the **Miscellaneous** section, check the box **Sepa Credit
-Transfer**.
+- Saint-Pierre-et-Miquelon;
+- Guernsey;
+- Jersey;
+- Isle of Man.
 
-Make sure to specify the IBAN account number (domestic account number
-won't work with SEPA) and the BIC (bank identifier code) on your bank
-journal.
+When paying a bill in Odoo, you can select SEPA mandates as a payment option. At the day's end,
+you can generate the SEPA file containing all bank wire transfers and send it to the bank.
 
-.. note::
-   By default, the payments you send using SEPA will use your company name as
-   initiating party name. This is what appears on the recipient's bank statement
-   in the **payment from** field. You can customize it in your company settings,
-   in the tab **Configuration**, under the **SEPA** section.
+By default, the file follows the SEPA Credit Transfer **'pain.001.001.03'** specifications. This is
+a well-defined standard among banks. However, for Swiss and German companies, other formats are used
+**'pain.001.001.03.ch.02'** for Switzerland and **'pain.001.003.03'** for Germany.
 
-.. image:: sepa/sepa01.png
-  :align: center
+Once the payments are processed by your bank, you can directly import the account statement in
+Odoo. The bank reconciliation process will seamlessly match the SEPA orders you sent to your bank
+with actual bank statements.
 
-Pay with SEPA
+Configuration
 =============
 
-Register your payments
-----------------------
-
-You can register a payment that is not related to a supplier bill. To do
-so, use the top menu :menuselection:`Purchases --> Payments`. Register your
-payment and select a payment method by Sepa Credit Transfer.
+Activate the SEPA Credit Transfer (SCT)
+---------------------------------------
 
-If it's the first time you pay this vendor, you will have to fill in the
-Recipient Bank Account field with, at least, the bank name, IBAN and BIC
-(Bank Identifier Code). Odoo will automatically verify the IBAN format.
+To pay suppliers with SEPA, you must activate the **SEPA Credit Transfer** setting.
+To do so, go to :menuselection:`Accounting app --> Configuration --> Settings -->
+Vendor Payments: --> SEPA Credit Transfer (SCT)`. By activating the setting and filling in with your
+company data, you will be able to add the SCT option when paying your vendor.
 
-For future payments to this vendor, Odoo will propose you automatically
-the bank accounts but you will be able to select another one or create a
-new one.
+.. note::
 
-If you pay a specific supplier bill, put the reference of the bill in
-the **memo** field.
+   According to the localization package installed, the **SEPA Direct Debit** and
+   **SEPA Credit Transfer** modules may be installed by default. If not, they need to be installed
+   :doc:`/applications/general/apps_modules`
```

Comment:
It's weird to append the title of a page to a sentence.
![image](https://user-images.githubusercontent.com/15896813/200594599-db9d1d62-d7eb-436b-a4cd-80a04847cd74.png)
Also, the trailing dot is missing.

Reply:
I changed the sentence so it looks less weird; I'm not sure what you mean by the missing trailing dot.